.github/actions/pr_notifier/pr_notifier.py

```diff
@@ -226,10 +228,33 @@ def post_to_oncall(client, unassigned_prs, out_slo_prs):
             text=(
                 "*Untriaged Issues* (please tag and cc area experts)\n<%s|%s>" %
                 (issue_link, issue_link)))
+        # On Monday, post the new oncall.
+        if datetime.date.today().weekday() == 0:
+            oncall = parse_calendar()
+            client.chat_postMessage(channel='#envoy-maintainer-oncall', text=(oncall))
+            client.chat_postMessage(channel='#general', text=(oncall))
     except SlackApiError as e:
         print("Unexpected error %s", e.response["error"])
 
 
+def parse_calendar():
+    e = requests.get(
```

Comment:
```suggestion
    response = requests.get(
```

`e` is usually reserved for errors as in `except Exception as e`